static/app/views/performance/landing/widgets/widgets/histogramWidget.tsx

```diff
@@ -61,7 +61,12 @@ export function HistogramWidget(props: Props) {
     <GenericPerformanceWidget<AreaDataType>
       {...props}
       Subtitle={() => (
-        <Subtitle>{t('Compared to last %s ', globalSelection.datetime.period)}</Subtitle>
+        <Subtitle>
+          {t(
+            'In the last %s ',
+            globalSelection.datetime.period ? globalSelection.datetime.period : 'period'
+          )}
```

Comment:
the string `period` won't be translated. Should this actually be more like


```suggestion
          {globalSelection.datetime.period
            ? t('In the last %s ', globalSelection.datetime.period)
            : t('In the last period')}
```

Reply:
Oh right 👍